osu.Game/Beatmaps/Formats/IHasComboColours.cs

```diff
@@ -8,6 +8,19 @@ namespace osu.Game.Beatmaps.Formats
 {
     public interface IHasComboColours
     {
-        List<Color4> ComboColours { get; set; }
+        /// <summary>
+        /// Retrieves the list of combo colours for presentation only.
+        /// </summary>
+        IReadOnlyList<Color4> ComboColours { get; set; }
+
+        /// <summary>
+        /// Adds combo colours to the list.
+        /// </summary>
+        void AddComboColours(params Color4[] colours);
```

Comment:
why was this added? i'm looking for reason for the added complexity in this PR.

Reply:
To allow passing default combo colours when trying to read via `ComboColours`, and avoid having 2 different values like this PR used to be: (`GetConfig(ComboColours)`, `SkinConfiguration.ComboColours`).